CONTRIBUTING.md

```diff
@@ -253,6 +262,11 @@ There are two ways you can do this:
 
 Each plugin/package has its own API Report which means you might see more then one file updated or created depending on your changes. These changes will then need to be committed as well.
 
+> [!WARNING]
+> If you encounter this error during `yarn build:api-reports` or the "check API reports and generate API reference" step in CI:
+> "The API Report for _plugin_ is not allowed to have warnings"
+>
+> Open the API report to view more details about the warning(s), resolve them, and regenerate the api reports with `yarn build:api-reports` before committing the changes.
```

Comment:
Can we add a blank line after this? Just makes it a bit easier to read when looking at the code version.